components/UI/ProjectsPanel/ProjectsPanel.tsx

```diff
@@ -0,0 +1,143 @@
+
+    const onSpace: KeyboardEventHandler<HTMLDivElement> = useCallback(
+        (event) => {
+            if (event.key === ' ') {
+                toggle();
+            }
+        },
+        [toggle],
+    );
+
+    useEffect(() => {
+        function onEsc(event: KeyboardEvent) {
+            if (event.key === 'Escape' && openMenu) {
+                close();
+            }
+        }
+
+        window.addEventListener('keyup', onEsc);
+
+        return () => {
+            window.removeEventListener('keyup', onEsc);
+        };
+    }, [close, openMenu]);
+
+    if (!activeProject) {
+        return null;
+    }
+
+    return (
+        <FocusLock disabled={!openMenu}>
+            <div
+                style={{ backgroundColor: theme.background, color: theme.color }}
+                className={classNames(styles.panel, {
+                    [styles.panel_active]: openMenu,
+                })}
+            >
+                <div onClick={close} className={styles.panel__overlay} />
+                <div className={styles.panel__modal}>
+                    <div className={styles.panel__scroller}>
+                        <div className={styles.projects}>
+                            <div className={styles.projects__title}>Экосистема Екатеринбурга</div>
+                            <ul className={styles.projects__list}>
+                                {projects.map((project) => (
+                                    <li className={styles.projects__item} key={project.shortTitle}>
+                                        <a
+                                            href={project.link}
+                                            className={styles.project}
+                                            key={project.shortTitle}
+                                        >
+                                            <img
+                                                src={project.logo}
+                                                className={styles.project__logo}
+                                                alt={project.shortTitle}
+                                            />
+                                            <div className={styles.project__title}>
+                                                {project.shortTitle}
+                                            </div>
+                                        </a>
+                                    </li>
+                                ))}
+                            </ul>
+                        </div>
+                        <div className={styles.panel__divider} />
+                    </div>
+                </div>
+                <div className={styles.panel__toggle}>
+                    <div
+                        tabIndex={0}
+                        role="button"
+                        onKeyDown={onSpace}
+                        className={styles.toggle}
+                        onClick={toggle}
```

Comment:
semantic police 👮‍♀️ 

Let's use `<button>` [1]

Reply:
img tag can't be inside the button. I used role button

Reply:
removed in new designs)